test/integration/integration_test.exs

```diff
@@ -49,13 +49,21 @@ defmodule KafkaEx.Integration.Test do
     assert consumer_group_update_interval == 30000
   end
 
-  test "create_worker provides a default consumer_group of 'kafka_ex'" do
```

Comment:
I suppose we should require `consumer_group` by default and it's either a binary (string) or an atom `:no_consumer_group` (you can chose a more fitting name here), so if it's not passed in `worker_init` or included in `config.exs` then we should blow up on creating the worker.